client/state/receipts/assembler.js

```diff
@@ -8,6 +8,7 @@ export function createReceiptObject( data ) {
 				meta: purchase.meta,
 				productId: purchase.product_id,
 				productSlug: purchase.product_slug,
+				productType: purchase.product_type,
```

Comment:
I think we want to keep this list of properties ordered alphabetically since it's easier to scan.